cocoapodsinstaller.go

```diff
@@ -0,0 +1,124 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/bitrise-io/go-steputils/v2/ruby"
+	"github.com/bitrise-io/go-utils/v2/command"
+	"github.com/bitrise-io/go-utils/v2/errorutil"
+	"github.com/bitrise-io/go-utils/v2/log"
+)
+
+// CocoapodsInstaller ...
+type CocoapodsInstaller struct {
+	rubyCmdFactory ruby.CommandFactory
+	logger         log.Logger
+}
+
+// NewCocoapodsInstaller ...
+func NewCocoapodsInstaller(rubyCmdFactory ruby.CommandFactory, logger log.Logger) CocoapodsInstaller {
+	return CocoapodsInstaller{
+		rubyCmdFactory: rubyCmdFactory,
+		logger:         logger,
+	}
+}
+
+// InstallPods ...
+func (i CocoapodsInstaller) InstallPods(podArg []string, podCmd string, podfileDir string, verbose bool) error {
+	if err := i.runPodInstall(podArg, podCmd, podfileDir, verbose); err == nil {
+		return nil
+	} else {
+		i.logger.Printf("")
+		i.logger.Warnf(errorutil.FormattedError(fmt.Errorf("Failed to install Pods: %w", err)))
```

Comment:
The caller in `main.go` also wraps the returned error in `errorutil.FormattedError()`, is this intentional?